<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QMainWindow" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #034da1;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>891</width>
      <height>721</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Shree Devanagari 714</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="data_frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>861</width>
       <height>261</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="credit_sum">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>391</width>
        <height>50</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,													stop: 0 #d4e8ff, stop: 1 #003d82 );
color: white;
border: 1px solid white;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="credit_term">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,													stop: 0 #d4e8ff, stop: 1 #003d82 );
color: white;
border: 1px solid white;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="credit_range">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,													stop: 0 #d4e8ff, stop: 1 #003d82 );
color: white;
border: 1px solid white;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="currency_4">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>120</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
     <widget class="QLabel" name="type">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>248</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Тип ежемесячных платежей</string>
      </property>
     </widget>
     <widget class="QLabel" name="term">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>61</y>
        <width>248</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Срок кредита</string>
      </property>
     </widget>
     <widget class="QLabel" name="range">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>121</y>
        <width>248</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Процентная ставка</string>
      </property>
     </widget>
     <widget class="QLabel" name="sum">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>2</y>
        <width>248</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Сумма кредита</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="months">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>60</y>
        <width>130</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>месяцев</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="years">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>88</y>
        <width>130</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>лет</string>
      </property>
     </widget>
     <widget class="QLabel" name="currency">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>0</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>руб.</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>190</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <layout class="QHBoxLayout" name="types">
       <item>
        <widget class="QRadioButton" name="a_type">
         <property name="font">
          <font>
           <family>Shree Devanagari 714</family>
           <pointsize>18</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Аннуитетные</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="d_type">
         <property name="font">
          <font>
           <family>Shree Devanagari 714</family>
           <pointsize>18</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Дифференциированные</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="count">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>300</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Shree Devanagari 714</family>
       <pointsize>20</pointsize>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #d4e8ff, stop: 1 #003d82 );
color: white;
border: 1px solid white;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #003d82, stop: 1 #d4e8ff);
}</string>
     </property>
     <property name="text">
      <string>Рассчитать</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>300</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Shree Devanagari 714</family>
       <pointsize>20</pointsize>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #d4e8ff, stop: 1 #003d82 );
color: white;
border: 1px solid white;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #003d82, stop: 1 #d4e8ff);
}</string>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
    <widget class="QFrame" name="result_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>380</y>
       <width>851</width>
       <height>331</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="result_percents">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>68</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,													stop: 0 #d4e8ff, stop: 1 #003d82 );
color: white;
border: 1px solid white;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="currency_1">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>8</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>руб.</string>
      </property>
     </widget>
     <widget class="QLabel" name="percents">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>69</y>
        <width>248</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Начисленные проценты</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="result_monthly_pay">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>8</y>
        <width>391</width>
        <height>50</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,													stop: 0 #d4e8ff, stop: 1 #003d82 );
color: white;
border: 1px solid white;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="result_total">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>128</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,													stop: 0 #d4e8ff, stop: 1 #003d82 );
color: white;
border: 1px solid white;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="total">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>129</y>
        <width>248</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Долг + проценты</string>
      </property>
     </widget>
     <widget class="QLabel" name="monthly_pay">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>248</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Ежемесячный платеж</string>
      </property>
     </widget>
     <widget class="QLabel" name="currency_2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>70</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>руб.</string>
      </property>
     </widget>
     <widget class="QLabel" name="currency_3">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>130</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Shree Devanagari 714</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>руб.</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="list_of_payments">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>391</width>
        <height>91</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,													stop: 0 #d4e8ff, stop: 1 #003d82 );
color: white;
border: 1px solid white;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="total_2">
         <property name="font">
          <font>
           <family>Shree Devanagari 714</family>
           <pointsize>18</pointsize>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>Список ежемесячных </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="total_3">
         <property name="font">
          <font>
           <family>Shree Devanagari 714</family>
           <pointsize>18</pointsize>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>платежей</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
